libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/VoiceInstructionLoader.java

```diff
@@ -66,6 +69,10 @@ public VoiceInstructionLoader(Context context, String accessToken, Cache cache)
   }
 
   public void cacheInstructions(List<String> instructions) {
+    if (!connectivityStatus.isConnected()) {
```

Comment:
Wondering if we could move this check up in the stack https://github.com/mapbox/mapbox-navigation-android/blob/a30d970639c058a7db1c890c16fdc1661c09d28d/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/VoiceInstructionCache.java#L28 and https://github.com/mapbox/mapbox-navigation-android/blob/a30d970639c058a7db1c890c16fdc1661c09d28d/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/VoiceInstructionCache.java#L52 and avoid all the logic / calculations that happen before calling `VoiceInstructionLoader#cacheInstructions` 🤔